recipes-example/template-dbus-service/template-dbus-service.bb

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+SRC_URI = "git@github.com:Pelagicore/template-dbus-service.git;protocol=ssh;branch=master"
+SRCREV = "f8a0e1bc987c2731b98ac69f55972ee9264ab518"
+
+inherit meson systemd
+
+DEPENDS += "glibmm gdbus-codegen-glibmm-native"
+
+S = "${WORKDIR}/git"
+
+# FIXME: Only include cmcli in -dev image?
```

Comment:
yes, fix me :)

Reply:
@martin-ejdestig I copy-pasted this from you, so I didn't know what it meant. Can we discuss this tomorrow?

Reply:
Name of binary is changed for template (`cmcli` is for `connectivity-manager`).

I was just not sure of whether we want to ship the CLI tool for connectivity-manager in !-dev images. Maybe we do? @sashko thinks yes? What do other people think?

Reply:
For `connectivity-manager` it is needed if one wants to verify that things work, at the moment. Since we have no UI. Could invoke D-Bus methods with `busctl`, I guess, but...

Reply:
This comment will definitely be removed and the cmcli will be replaced with the "tdbscli", which is used for the template-dbus-service. It is a candidate for removal for production, but not for development images.